[image_view] Build error on ArchLinux because of not found ImageViewConfig.h

Firstly reported at: https://github.com/ros-perception/image_pipeline/commit/6aa8062e2635306be3322f611e010ea872535346#commitcomment-16596255

```
Scanning dependencies of target image_view_exe
[ 70%] Building CXX object CMakeFiles/image_view_exe.dir/src/nodes/image_view.cpp.o
/tmp/yaourt-tmp-username/aur-ros-indigo-image-view/src/image_view/src/nodes/image_view.cpp:34:40: fatal error: image_view/ImageViewConfig.h: No such file or directory
compilation terminated.
```

--- a/image_view/cfg/ImageView.cfg
+++ b/image_view/cfg/ImageView.cfg
@@ -1,13 +1,7 @@
 #! /usr/bin/env python
 
 PACKAGE='image_view'
-try:
-    import imp
-    imp.find_module(PACKAGE)
-    from dynamic_reconfigure.parameter_generator_catkin import *
-except:
-    import roslib; roslib.load_manifest(PACKAGE)
-    from dynamic_reconfigure.parameter_generator import *
+from dynamic_reconfigure.parameter_generator_catkin import *
 
 gen = ParameterGenerator()
 

